toolkit/resources/manifests/package/pkggen_core_aarch64.txt

```diff
@@ -208,7 +208,7 @@ pcre-8.45-2.cm2.aarch64.rpm
 pcre-libs-8.45-2.cm2.aarch64.rpm
 lua-5.4.3-5.cm2.aarch64.rpm
 lua-libs-5.4.3-5.cm2.aarch64.rpm
-mariner-rpm-macros-2.0-20.cm2.noarch.rpm
+mariner-rpm-macros-2.0-21.cm2.noarch.rpm
 mariner-check-macros-2.0-20.cm2.noarch.rpm
```

Comment:
`mariner-check-macros` will need its release number bumped in the manifests as well- it's a subpackage of `mariner-rpm-macros`.

Reply:
Ahh, apologies, I missed that. Fixed